README.md

```diff
@@ -1,49 +1,52 @@
 # Lotus Eth JSON-RPC integration tests
 
-This project holds a suite of integration tests for the Ethereum JSON-RPC API
-built in Lotus with the Filecoin EVM runtime.
+This project holds a suite of integration tests for the Ethereum JSON-RPC API built in [Lotus](https://github.com/filecoin-project/lotus) with the Filecoin EVM runtime.
 
-This repo is the test runner for all the project repo in the `extern/` folder.
+This repo runs some basic tests direclty against ethers.js and web3.js libraries, and also runs deployment and tests in the [fevm-hardhat-kit](https://github.com/filecoin-project/fevm-hardhat-kit) (deploy only), [openzeppelin-contracts](https://github.com/filecoin-project/openzeppelin-contracts) and [fevm-uniswap-v3-core](https://github.com/DigitalMOB2/fevm-uniswap-v3-core) repositories, each of which use hardhat to deploy and test smart contracts on the Filecoin EVM.
+
+A custom version of Lotus is built to run these tests, which is included in the [`node`](./node) directory. Each build pulls the current `master` branch of Lotus, so the tests are always run against the latest version of Lotus.
+
+See [`Makefile`](./Makefile) for more details on how the tests are set up and executed.
 
 ## Running tests
 
-### Build the local lotus
+### Build the local lotus-runner
 
-1. Go through the [Lotus installation](https://lotus.filecoin.io/lotus/install/prerequisites/) guide to install the dependencies
-2. Install node dependencies by running `npm install`
-3. cd `./node` and run `make node`
-4. Run the node in `./bin/node`
+1. Ensure all dependencies are installed as described for a [Lotus installation](https://lotus.filecoin.io/lotus/install/prerequisites/)
+2. Run `make build-lotus-runner`
 
-### Prepare the .env file
+### Install the dependencies
 
-1. echo -n DEPLOYER_PRIVATE_KEY=0x > .env
-2. openssl rand -hex 32 >> .env
-3. echo -n USER_1_PRIVATE_KEY=0x >> .env
-4. openssl rand -hex 32 >> .env
-5. Copy the .env file into ./extern/fevm-hardhat-kit/, ./extern/openzeppelin-contracts/, ./extern/fevm-uniswap-v3-core/
+Run `make install`
+
+This will install dependencies for each of:
+1. The kit to interact with the lotus-runner (`make install-kit`)
+2. The local ethers.js and web3.js library tests (`make install-libs`)
+3. fevm-hardhat-kit (`make install-fevm-hardhat`)
+4. openzeppelin-contracts (`make install-openzeppelin`)
+5. fevm-uniswap-v3-core (`make install-uniswap-v3-core`)
 
 ### Run each test project
 
-1. ethers.js and web3.js (root folder)
-   1. npm install
-   2. npx hardhat --network itest test
-2. ./extern/fevm-hardhat-kit
-   1. change the `DEPLOYER_PRIVATE_KEY` in .env to `PRIVATE_KEY`
-   2. yarn
-   3. npx hardhat --network itest deploy
-3. ./extern/openzeppelin-contracts 
-   1. npm install
-   2. npx hardhat --network itest test
-4. ./extern/fevm-uniswap-v3-core
-   1. yarn
-   2. npx hardhat --network itest test
+1. Run the local lotus-runner: `make start-lotus-runner`
```

Comment:
we need to run `make build-lotus-runner`
- we can either include that in install or include that in the documentation here

Reply:
added, but it is already listed in the build step above

Reply:
actually, I just set up a build dependency to make sure it's built, so it's easier now